senator/utility.py

```diff
@@ -42,6 +54,7 @@ def load_data(file_path: str) -> np.ndarray:
 
 def parse_data(data: np.ndarray) -> tuple[np.ndarray, np.ndarray]: 
```

Comment:
Špatně type hints :) To mi připomíná, další rozšíření je `mypy`, který dělá automatický checking typů a upozorní tě když děláš něco divnýho. @juliakrizanova máš s tím zkušenosti ve VS code?

Reply:
To tam nejspíš také zůstalo z nějaké předchozí iterace. Počkám, co poradí mamka =D

Reply:
S tymto skusenosti nemam, ale znie to ako uzitocna vec. Pozriem ako to spojazdnit vo VScode a dam vediet:D